### ISSUE TITLE
unknown image for calendar entries

### ISSUE THREAD

**Issue by psilo909** [Martin-Gleiss/smartvisu#354]
combined with commit https://github.com/Martin-Gleiss/smartvisu/commit/4980fc131a45c97367695076f5d0de501cb0453b, the default language ini file must be also updated:

https://github.com/Martin-Gleiss/smartvisu/blob/develop/lang/de.ini#CL279
here we have
```
default_img_list[icon] = ''
default_img_list[color] = "rgb(32, 178, 170)"
```

in this case, the calendar.js widget will run into the else:
```
if (typeof(sv_lang.calendar_event_format.default_img_list) === undefined){
   entry.icon = "pages/base/pics/trans.png";
   entry.color = 'transparent';
}else{
   entry.icon = sv_lang.calendar_event_format.default_img_list.icon;
   entry.color = sv_lang.calendar_event_format.default_img_list.color;
}
```

and set an empty string as image. in result, it shows up like this:
![unknown](https://user-images.githubusercontent.com/15831643/85209117-96a11880-b335-11ea-8565-a8b4a6a56ca6.png)
![image](https://user-images.githubusercontent.com/15831643/85209046-21354800-b335-11ea-8349-81c0b682bd0a.png)

if changed to
```
default_img_list[icon] = 'pages/base/pics/trans.png'
default_img_list[color] = "transparent"
```
all is fine.

![fine](https://user-images.githubusercontent.com/15831643/85209131-b0daf680-b335-11ea-89ed-236547bdf11c.png)

PS: I suppose also checking for an empty string in the formerly mentioned javascript snippet (and not only for undefined) would also do the trick.

**Comment by wvhn** [comment]
Where does the " == $0" behind the image tag come from?
In my setup w/ Apache and Firefox, the image tag looks like this:
<img class="icon" style="background: rgb(32, 178, 170) none repeat scroll 0% 0%;" src="">
So all ist fine here.
Nevertheless I have integrated an additional check for an empty string to get trans.png back.

**Comment by wvhn** [comment]
solved with #356
